laplace/utils.py

```diff
@@ -183,6 +186,6 @@ def block_diag(blocks):
     p_cur = 0
     for block in blocks:
         p_block = block.shape[0]
-        M[p_cur:p_cur+p_block, p_cur:p_cur+p_block] = block
+        M[p_cur : p_cur + p_block, p_cur : p_cur + p_block] = block
```

Comment:
I think the additional spaces here make it harder to parse, but maybe I'm just unfamiliar to this style.